signer/tests/regtest/mod.rs

```diff
@@ -0,0 +1,363 @@
+use bitcoin::absolute::LockTime;
+use bitcoin::key::TapTweak;
+use bitcoin::sighash::Prevouts;
+use bitcoin::sighash::SighashCache;
+use bitcoin::transaction::Version;
+use bitcoin::Address;
+use bitcoin::AddressType;
+use bitcoin::Amount;
+use bitcoin::CompressedPublicKey;
+use bitcoin::EcdsaSighashType;
+use bitcoin::Network;
+use bitcoin::OutPoint;
+use bitcoin::PublicKey;
+use bitcoin::ScriptBuf;
+use bitcoin::Sequence;
+use bitcoin::TapLeafHash;
+use bitcoin::TapNodeHash;
+use bitcoin::TapSighashType;
+use bitcoin::Transaction;
+use bitcoin::TxIn;
+use bitcoin::TxOut;
+use bitcoin::Witness;
```

Comment:
What are your thoughts on this import style?
```rust
use bitcoin::{Address, AddressType, Amount, CompressedPublicKey, ...}
```

Reply:
I used to prefer the default style, but now I think this `item` style is better. It makes diffs easier to see/read and makes resolving any git conflicts simpler. I know it's the default style so I'm okay switching to it.

Reply:
Oh, no this is a reasonable case for it, I kind of like it